infrasec/tf/atlantis.md

```diff
@@ -1,14 +1,219 @@
 # Atlantis Truss
 
-We're beginning to use Atlantis throughout projects at Truss. There's more to come, but for now here are some helpful things to know.
+## Step-by-Step Atlantis Implementation
+
+### Where to start
+
+First off, there are [3 ways to configure Atlantis](https://www.runatlantis.io/docs/configuring-atlantis.html). Step one is looking at your own configuration and deciding which way to do the thing.
+
+tl;dr from [Atlantis docs](https://www.runatlantis.io/docs/server-configuration.html)-
+
+```bash
+Configuration to atlantis server can be specified via command line flags, environment variables, a config file or a mix of the three.
+```
+
+In the [legendary waddle](https://github.com/trussworks/legendary-waddle) repo, for example, we used a combination of [environment variables](https://github.com/trussworks/legendary-waddle/blob/master/trussworks-prod/atlantis-prod/main.tf#L85-L102) and [repo-level `atlantis.yaml` files](https://github.com/trussworks/legendary-waddle/blob/master/atlantis.yaml).
```

Comment:
When linking directly to specific lines, can you use the permalink so that when the upstream file changes, the docs here still point to the original lines?

Reply:
Good idea. Yeah I need to update/remove a lot of these links that I crammed in here for my own use.